app/helpers/custom_form_error_helper.rb

```diff
@@ -0,0 +1,14 @@
+module CustomFormErrorHelper
+  def errors(object, attribute)
+    error_messages = object.try(:errors).try(:full_messages_for, attribute)
+
+    if error_messages.is_a?(Array)
+      error_contents = create_error_div(error_messages)
+    end
+    error_contents || ""
+  end
+
+  def create_error_div(error_messages)
+    raw "<small class=""help-block"">#{error_messages.first}</small>"
```

Comment:
__[RuboCop]__ Lint/ImplicitStringConcatenation: Combine "<small class=" and "help-block" into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma. ([link](https://sider.review/gh/repos/47168038/pulls/585#issue-4232495))

<sub>posted by Sider</sub>